docs/source/concepts/ledger-topologies.rst

```diff
@@ -0,0 +1,179 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _daml-ledger-topologies:
+
+DAML Ledger Topologies
+######################
+
+The Ledger API provides parties with an abstraction of a virtual shared ledger, visualized as follows.
+
+.. image:: ./ledger-model/images/da-ledger-model.svg
+   :width: 80%
+   :align: center
+
+The real-world topologies of actual ledger implementations differ significantly, however.
+The topologies can impact both the functional and non-functional properties of the resulting ledger.
+This document:
+
+1. Provides one useful categorization of the existing implementations' topologies.
+   The categorization is not the only one possible, and it is not always clear-cut.
+   Its main aim is to group the implementations according to their high-level properties.
+
+2. Describes the general ledger properties of each category.
+
+
+.. _global-state-topologies:
+
+Global State Topologies
+***********************
+
+.. _trust-domain:
+
+We call a system operated by a single real-world entity a **trust domain**.
+In global state topologies, there exists at least one trust domain (and possibly more) whose systems contain a physical copy of the entire virtual shared ledger that is accessible through the API.
+
+.. _fully-centralized-ledger:
+
+The Fully Centralized Ledger
+============================
+
+The simplest global state topology is the one where the virtual shared ledger is implemented through a single machine containing a physical copy of the shared ledger.
+
+.. image:: ./images/ledger-topologies/physical-shared-ledger.svg
+   :width: 80%
+   :align: center
+
+The :ref:`DAML Sandbox <sandbox-manual>` uses this topology.
+While simple, this topology has certain downsides:
+
+- it provides no scaling
+
+- it is not highly available
+
+- the real-world entity operating the physical shared ledger is fully trusted with preserving the ledger's integrity
+
+- the real-world entity operating the physical shared ledger has full insight into the entire ledger, and is thus fully trusted with privacy
```

Comment:
`real-world entity` vs. `single physical copy` vs. `trust domain`. Consider defining `operating entity` as the entity operating `trust domain` use that in the definition of `Fully Centralized Ledger` topology